telegram/_message.py

```diff
@@ -826,11 +826,19 @@ def id(self) -> int:  # pylint: disable=invalid-name
     def link(self) -> Optional[str]:
         """:obj:`str`: Convenience property. If the chat of the message is not
         a private chat or normal group, returns a t.me link of the message.
+
+            .. versionchanged:: NEXT.VERSION
+                Added support for topics.
         """
         if self.chat.type not in [Chat.PRIVATE, Chat.GROUP]:
             # the else block gets rid of leading -100 for supergroups:
             to_link = self.chat.username if self.chat.username else f"c/{str(self.chat.id)[4:]}"
-            return f"https://t.me/{to_link}/{self.message_id}"
+            baselink = f"https://t.me/{to_link}/{self.message_id}"
+
+            if self.is_topic_message and self.message_thread_id:
+                baselink += f"?thread={self.message_thread_id}"
```

Comment:
```suggestion
                baselink = f"{baselink}?thread={self.message_thread_id}"
```
I heard f-strings are faster than addition, and we have an f-string here anyway